src/js/components/pages/TabsForGoodSettings.jsx

```diff
@@ -123,7 +124,7 @@ const CharitySelectBox = ({charity, deselect, padAmount3D=150, className}) => {
 			{charity ? <>
 				<CharityLogo style={{maxWidth:"100%", width: "100%", transform: `translateZ(${elementHeight}px)`}} charity={charity} key={charity.id} className="p-2 mb-5 mt-5 w-75"/>
 				{deselect ? <a className="btn btn-primary thin position-absolute" style={{bottom:20, left:"50%", transform:"translateX(-50%)"}} onClick={() => deselectCharity(charity)}>Deselect</a>
-					: <a className="btn btn-transparent fill thin position-absolute" style={{bottom:20, left:"50%", transform:"translateX(-50%)"}} onClick={() => selectCharity(charity)}>Select</a>}
+					: <a className="btn btn-transparent fill thin position-absolute" style={{bottom:20, left:"50%", transform:"translateX(-50%)"}} onClick={() => setSelectedCharityId(charity)}>Select</a>}
```

Comment:
The charity prop passed here is a charity object I think, but is being used in the function as a charity ID

Reply:
Good spot - thanks